metadata_service/api/user.py

```diff
@@ -50,15 +52,15 @@ def get(self, user_id: str) -> Iterable[Union[Mapping, int, None]]:
             return {'message': 'User id {} does not exist'.format(user_id)}, HTTPStatus.NOT_FOUND
 
 
-class UserFollowAPI(Resource):
+class UserFollowsAPI(Resource):
```

Comment:
why do we need to add new class for get instead of putting get/put etc under a same resource?

Reply:
I think it is described in the PR that was linked in the issue you raised: https://github.com/lyft/amundsensearchlibrary/pull/60 (I was seeing the same issue in metadata service where some URLs were invalid for some methods)

> [...] However, the Swagger documentation is looking at all the endpoints and methods setup in Flask, so this bug was causing the Swagger documentation to give users bad information. 😞

> Why is this happening?

> These lines of code in the service init file are the problem. This code is adding the resource to Flask and allowing all methods to be accessed via both URLs. However, both URLs are not supported by our methods.

In summary, each class only has "one type of URL"

Reply:
ah, forgot about that:) thanks.